libs/feature/search/src/lib/results-list/results-list.container.component.html

```diff
@@ -6,6 +6,22 @@
     [favoriteTemplate]="favoriteToggle"
     (mdSelect)="onMetadataSelection($event)"
   ></gn-ui-results-list>
+
+  <ng-container *ngIf="(facade.isLoading$ | async) === false">
+    <div class="show-more" *ngIf="showMore === 'button'">
+      <gn-ui-button type="primary" (click)="onShowMore()">
+        <span translate>results.showMore</span>
+      </gn-ui-button>
+    </div>
+    <div
+      class="show-more absolute left-[0px] bottom-[0px] right-[0px] height-[2px]"
```

Comment:
you have `inset` classes for that ! maybe only in v3 idk

Reply:
I've added a commit that restores the possibility to do `inset-0` and similar, I've always wondered why we wouldn't be able to do that but it was actually just because of the TW config. So I went ahead and also simplified a lot of places by using TW classes instead of CSS with `left: 0`, `top: 0` etc.

@fgravin can you let me know if you think that's OK?